src/driver.ts

```diff
@@ -534,10 +615,25 @@ export abstract class CMakeDriver implements vscode.Disposable {
       flags.push(...util.objectPairs(this._kit.cmakeSettings).map(([key, val]) => _makeFlag(key, util.cmakeify(val))));
     }
 
+    // Expand all flags
     const final_flags = flags.concat(settings_flags);
-    log.trace('CMake flags are', JSON.stringify(final_flags));
+    const expanded_flags_promises = final_flags.map(
+        async (value: string) => this.expandString(value, await this.getExpandedConfigureEnvironment()));
```

Comment:
If you lift `this.getExpandedConfigureEnvironment` into a variable before you expand the flags, we'll only call the function once. `getExpandedConfigureEnvironment()` could be expensive, depending on what they have in their configure environment.